azure-mgmt-search/src/main/java/com/microsoft/azure/management/search/SearchServices.java

```diff
@@ -0,0 +1,200 @@
+   * @param searchServiceName The name of the Azure Search service associated with the specified resource group.
+   * @param name The name of the new query API key.
+   * @throws IllegalArgumentException thrown if parameters fail the validation
+   * @throws CloudException thrown if the request is rejected by server
+   * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent
+   * @return the List&lt;QueryKey&gt; object if successful.
+   */
+  QueryKey createQueryKey(String resourceGroupName, String searchServiceName, String name);
+
+  /**
+   * Regenerates either the primary or secondary admin API key. You can only regenerate one key at a time.
+   *
+   * @param resourceGroupName The name of the resource group within the current subscription. You can obtain this value from the Azure Resource Manager API or the portal.
+   * @param searchServiceName The name of the Azure Search service associated with the specified resource group.
+   * @param name The name of the new query API key.
+   * @throws IllegalArgumentException thrown if parameters fail the validation
+   * @return the observable to the List&lt;QueryKey&gt; object
+   */
+  Observable<QueryKey> createQueryKeyAsync(String resourceGroupName, String searchServiceName, String name);
+
+  /**
+   * Deletes the specified query key. Unlike admin keys, query keys are not regenerated. The process for
+   * regenerating a query key is to delete and then recreate it.
+   *
+   * @param resourceGroupName The name of the resource group within the current subscription. You can obtain this value from the Azure Resource Manager API or the portal.
+   * @param searchServiceName The name of the Azure Search service associated with the specified resource group.
+   * @param key The query key to be deleted. Query keys are identified by value, not by name.
+   * @throws IllegalArgumentException thrown if parameters fail the validation
+   * @throws CloudException thrown if the request is rejected by server
+   * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent
+   */
+  void deleteQueryKey(String resourceGroupName, String searchServiceName, String key);
+
+  /**
+   * Deletes the specified query key. Unlike admin keys, query keys are not regenerated. The process for
+   * regenerating a query key is to delete and then recreate it.
+   *
+   * @param resourceGroupName The name of the resource group within the current subscription. You can obtain this value from the Azure Resource Manager API or the portal.
+   * @param searchServiceName The name of the Azure Search service associated with the specified resource group.
+   * @param key The query key to be deleted. Query keys are identified by value, not by name.
+   * @throws IllegalArgumentException thrown if parameters fail the validation
+   * @return the Observable to {@link Completable} object if successful.
```

Comment:
we used to use `@link` in the past, but are now removing it because it doesn't convert properly to C#